zulipterminal/ui_tools/views.py

```diff
@@ -1929,6 +1942,57 @@ def keypress(self, size: urwid_Size, key: str) -> str:
         return super().keypress(size, key)
 
 
+class ReadReceiptView(PopUpView):
+    def __init__(
+        self,
+        controller: Any,
+        message: Message,
+        topic_links: "OrderedDict[str, Tuple[str, int, bool]]",
+        message_links: "OrderedDict[str, Tuple[str, int, bool]]",
+        time_mentions: List[Tuple[str, str]],
+        title: str,
+    ) -> None:
+        self.controller = controller
+        self.message = message
+        self.topic_links = topic_links
+        self.message_links = message_links
+        self.time_mentions = time_mentions
+        max_cols, max_rows = controller.maximum_popup_dimensions()
+
+        # Get rendered message header and footer
+        msg_box = MessageBox(message, controller.model, None)
```

Comment:
What is the purpose of this?

Reply:
Ah I realised that we don't need a message header or footer for this popup but forgot to remove that code line. Will do that.